### PR TITLE
Update gs-authentication.md

### PR DESCRIPTION
**Pull request by chirag-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [x] Bug fix or improvement
 

## Summary
 Fixes typo
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/get-started/authentication/gs-authentication.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Guest user | Resources with `anonymous` permission.
 
 <h3 id="acl-webapi-relation">Relation between acl.xml and webapi.xml</h3>
 The <code>acl.xml</code> file defines the access control list (ACL) for a given {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %}. It defines available set of permissions to access the resources.
-`acl.xml` files across all Magento modules are consolidated to build an ACL tree which is used to select allowed {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %} role resources or third party Integration's access (System > {% glossarytooltip 55774db9-bf9d-40f3-83db-b10cc5ae3b68 %}Extension{% endglossarytooltip %} > Integration > Add New Integration > Available APIs).
+`acl.xml` files across all Magento modules are consolidated to build an ACL tree which is used to select allowed {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %} role resources or third party Integration's access (<b>System > Extension > Integration > Add New Integration > Available APIs</b>).
```
Comment (shrielenee) [inline guides/v2.1/get-started/authentication/gs-authentication.md]: We use markdown for text formatting. Please replace `<b>` with `**` surrounding what you want to be bold.

Comment (shrielenee) [reply guides/v2.1/get-started/authentication/gs-authentication.md]: Any other places that you can convert HTML formatting to markdown formatting is also good!

**Comment by chirag-wagento** [comment]
@shrielenee  As per your advice i have committed new code please check it.

Thanks

**Comment by shrielenee** [comment]
Great job! Thanks @chirag-wagento!

**Pull request merged**